openlibrary/plugins/openlibrary/js/ol.analytics.js

```diff
@@ -23,5 +25,26 @@ export default function initAnalytics() {
                 cache_bust: Math.random()
             });
         }
+
+        vs = window.archive_analytics.get_data_packets();
+        for (i in vs) {
+            vs[i]['cache_bust']=Math.random();
+            vs[i]['server_ms']=$('.analytics-stats-time-calculator').data('time');
+            vs[i]['server_name']='ol-web.us.archive.org';
+            vs[i]['service']='ol';
+        }
+        if (window.flights){
+            window.flights.init();
+        }
+        if ($('.more_search').size()>0) {
```

Comment:
Note, size is a deprecated jquery function. We should make sure to use length in a follow up.

Reply:
Right, noted. 
As written in my first comment, I'm still confused thought, there is no mention of a `more_search` class or `window.flights` anywhere else in the code, could you verify? Maybe it is redundant code?

Reply:
It's very possible. I'm out and about right now but when i have my computer I can take a closer look. Thanks for flagging this potentially dead code. That's really helpful!

Reply:
I am pretty sure you are right, the more_search selector  matches nothing. If you could  submit a follow up patch to remove this line, that would be super helpful!

Reply:
Gotcha, opening a PR soon.